### DISCUSSION TITLE
umi4，Duplicate namespace in models

### DISCUSSION THREAD

**Discussion by luanwulin** [umijs/umi#8364] [Q&A]
4.0.5版本依然会报错
 Error: Duplicate namespace in models: editor.components.left.model,editor.components.left.model, editor.components.main.actions.model, editor.components.main.actions.model

models都声明了namespace键值，但是umi4好像还是取得文件名

类似这个issue [dva开启，pages下 model.ts文件，提示Error: Duplicate namespace in models:错误](https://github.com/umijs/umi-next/issues/844)

**Comment by fz6m** [comment]
给一个复现 demo 看看

**Comment by luanwulin** [comment]
@fz6m 已经上传了 [demo](https://github.com/luanwulin/umi4-bug)

还是会报错
![image](https://user-images.githubusercontent.com/16553854/177114616-98201ad6-af00-442b-bedd-9658e2b19009.png)

**Reply by sorrycc** [reply]
检查下，为啥会有两个 `editor.model`？

**Reply by luanwulin** [reply]
@sorrycc 难道是因为ts被转换成js，同时有两个model文件么？升级到umi4过后，只要跑dev就会把ts的js和map文件都给生成出来
![image](https://user-images.githubusercontent.com/16553854/177264004-dc498b75-fd14-419b-b01a-35f22df78ac9.png)

**Reply by litang1010** [reply]
解决了吗，大佬
